packages/react-core/src/components/Drawer/examples/Drawer.md

```diff
@@ -22,91 +23,119 @@ section: components
 ### Basic
 
 ```ts file="./DrawerBasic.tsx"
+
 ```
 
 ### Panel on right
 
 ```ts file="./DrawerPanelRight.tsx"
+
 ```
 
 ### Panel on left
 
 ```ts file="./DrawerPanelLeft.tsx"
+
 ```
 
 ### Panel on bottom
 
 ```ts file="./DrawerPanelBottom.tsx"
+
 ```
 
 ### Basic inline
 
 ```ts file="./DrawerBasicInline.tsx"
+
 ```
 
 ### Inline panel on right
 
 ```ts file="./DrawerInlinePanelRight.tsx"
+
 ```
 
 ### Inline panel on left
 
 ```ts file="./DrawerInlinePanelLeft.tsx"
+
 ```
 
 ### Stacked content body elements
 
 ```ts file="./DrawerStackedContentBodyElements.tsx"
+
 ```
 
 ### Modified content padding
 
 ```ts file="DrawerModifiedContentPadding.tsx"
+
 ```
 
 ### Modified panel padding
 
 ```ts file="DrawerModifiedPanelPadding.tsx"
+
 ```
 
 ### Additional section above drawer content
 
 ```ts file="DrawerAdditionalSectionAboveContent.tsx"
+
 ```
 
 ### Static drawer
 
 **Note:** For mobile viewports, all drawer variants behave the same way. At the `md` breakpoint, or where `.pf-m-static{-on-[lg, xl, 2xl]}` is applied, the `static drawer` variant’s `close button` is automatically hidden because the drawer panel doesn’t close by design.
 
 ```ts file="DrawerStatic.tsx"
+
 ```
 
 ### Breakpoint
 
 ```ts file="DrawerBreakpoint.tsx"
+
 ```
 
 ### Resizable on right
 
 ```ts file="DrawerResizableOnRight.tsx"
+
 ```
 
 ### Resizable on left
 
 ```ts file="DrawerResizableOnLeft.tsx"
+
 ```
 
 ### Resizable on bottom
 
 ```ts file="DrawerResizableOnBottom.tsx"
+
 ```
 
 ### Resizable on inline
 
 ```ts file="DrawerResizableOnInline.tsx"
+
 ```
 
 ### Panel with secondary background
 
 ```ts file="DrawerSecondaryBackground.tsx"
+
+```
+
+### With focus trap
+
+Use the `focusTrap` property to enable and customize a focus trap on the `<DrawerPanelContent>`. Enabling a focus trap with `focusTrap.enabled` will also automatically place focus on the first focusable element when the drawer panel expands, and return focus to the previously focused element when it collapses.
```

Comment:
```suggestion
A focus trap contains focus within an overlay element. When a focus trap is enabled on an element, a user can only interact with elements inside of the focused area until the focus trap is closed or deactivated.  

To enable and customize a focus trap on a drawer panel, apply the `focusTrap` property to the `<DrawerPanelContent>` component. Enabling a focus trap with `focusTrap.enabled` will also automatically place focus on the first focusable element when the drawer panel expands, and return focus to the previously focused element when it collapses.
```

Does this addition sound accurate and useful? Thought it may be good to briefly describe what a focus trap is, but lmk if not (I scrapped together an understanding from a little googling).

Reply:
It might be worth mentioning. Considering other components use a focus trap, would it be better to have it in a single place that can be referenced? We do have content on [trapping focus](https://www.patternfly.org/accessibility/product-development-guide#trapping-focus) on the PF site, though it's a bit more specific to Modal and Popover.

For the example description, what do you think about, "When a focus trap is enabled on an element, a user will only be able to interact with the contents of that element until the focus trap is closed or deactivated." as the first line (followed by the "To enable and customize a focus trap..." content)?

Reply:
Ahh thanks for the extra info! Since we have that section on trapping focus in the accessibility docs, what if we just link to that doc on the first instance of "focus trap"? If that trapping focus info would benefit from being expanded we could always do that in a followup issue (doing a content review/audit of the accessibility docs is already on my to-do list as part of the wider site content audit)?

I like your suggestion for reframing the content!